.github/workflows/rpcTest.yml

```diff
@@ -0,0 +1,37 @@
+name: RPC test
+
+on:
+  pull_request:
+    branches:
+      - master
+
+  push:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16.x]
+    steps:
+    - uses: actions/checkout@v2
+
+    - run: wget -q $(curl -s https://api.github.com/repos/AstarNetwork/Astar/releases/latest | grep browser_download_url | grep "ubuntu-x86_64" | cut -d '"' -f 4)
+
+    - run: tar -xvf astar-collator-*-ubuntu-x86_64.tar.gz
+    
+    - run: chmod +x ./astar-collator
```

Comment:
We should be able to start tests for a selected tag or release, I assume.
The most important thing is that regression testing is made easier.

This is good for testing but not for merge.

Reply:
Added workflow_dispatch trigger in rpc test. Workflow can be run for any release with a release tag input.
![Screenshot from 2022-07-07 15-28-46](https://user-images.githubusercontent.com/874046/177785570-330c6613-237d-4190-bfed-69876209daf6.png)